src/Core/Context.cpp

```diff
@@ -118,14 +120,36 @@ void Context::Setup() {
 }
 
 void Context::Update() {
-    Util::Time::Update();
     Util::Input::Update();
     SDL_GL_SwapWindow(m_Window);
     glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 
-    constexpr double frameTime =
-        FPS_CAP != 0 ? 1000 / static_cast<double>(FPS_CAP) : 0;
-    SDL_Delay(static_cast<Uint32>(frameTime - Util::Time::GetDeltaTime()));
+    constexpr ms_t frameTime =
+        FPS_CAP != 0 ? static_cast<float>(1000.0F / FPS_CAP) : 0;
```

Comment:
sorry my admired sonarcloud ![Prayge](https://cdn.7tv.app/emote/60aec2196cfcffe15f4e4f93/1x.webp)
```suggestion
        FPS_CAP != 0 ? 1000.0F / FPS_CAP : 0;
```